src/components/SelectToken.vue

```diff
@@ -67,10 +67,7 @@ export default class SelectToken extends Mixins(TranslationMixin, DialogMixin, L
   @Prop({ default: () => false, type: Boolean }) readonly notNullBalanceOnly!: boolean
 
   @Getter('assets', { namespace }) assets!: Array<Asset>
-  @Action('getAssets', { namespace }) getAssets
```

Comment:
We should call it any time when we open this dialog, because this list may be changed